.gitignore

```diff
@@ -6,6 +6,7 @@ ignored-assets
 main.wasm
 .parcel-cache
 .vscode/*.log
+go.sum
```

Comment:
cc @adamreese — is adding `go.sum` to `.gitignore` the right thing to do here?
I mainly want to make sure we're not polluting the repo with changes here.